internal/processor/lidarr.go

```diff
@@ -38,16 +38,28 @@ func (s Service) lidarr(ctx context.Context, cfg *domain.ArrConfig, dryRun bool,
 
 		l.Debug().Msgf("updating filter: %v", filterID)
 
-		if !dryRun {
-			f := autobrr.UpdateFilter{MatchReleases: joinedTitles}
+		if !cfg.MatchRelease {
+			if !dryRun {
+				f := autobrr.UpdateFilter{Albums: joinedTitles}
+
+				if err := brr.UpdateFilterByID(ctx, filterID, f); err != nil {
+					l.Error().Err(err).Msgf("something went wrong updating music filter: %v", filterID)
+					continue
+				}
+			}
+
+			l.Debug().Msgf("successfully updated filter: %v", filterID)
+		} else {
+			if !dryRun {
+				f := autobrr.UpdateFilter{MatchReleases: joinedTitles}
 
-			if err := brr.UpdateFilterByID(ctx, filterID, f); err != nil {
-				l.Error().Err(err).Msgf("something went wrong updating album filter: %v", filterID)
-				continue
+				if err := brr.UpdateFilterByID(ctx, filterID, f); err != nil {
+					l.Error().Err(err).Msgf("something went wrong updating music filter: %v", filterID)
+					continue
+				}
 			}
 		}
```

Comment:
This and the other similar functions could be simplified with this:

```golang
f := autobrr.UpdateFilter{Albums: joinedTitles, MatchReleases: ""}

if cfg.MatchRelease {
	f = autobrr.UpdateFilter{MatchReleases: joinedTitles}
} 

if !dryRun {
	if err := brr.UpdateFilterByID(ctx, filterID, f); err != nil {
		l.Error().Err(err).Msgf("something went wrong updating music filter: %v", filterID)
		continue
	}
}

l.Debug().Msgf("successfully updated filter: %v", filterID)
```

Would improve readability and make the code just a tad bit cleaner.

And with the first `UpdateFilter` like this below we would override the `MatchReleases` to be an empty string if previously was set, so users don't need to manually empty that field when updating to new versions.

```golang
f := autobrr.UpdateFilter{Albums: joinedTitles, MatchReleases: ""}
```

Reply:
Thanks, fixed 👍